iocage/lib/events.py

```diff
@@ -176,8 +176,11 @@ def duration(self) -> typing.Optional[float]:
             return None
 
     def _update_message(self, **kwargs) -> None:
-        if "message" in kwargs:
-            self.message = kwargs["message"]
+        for key in kwargs.keys():
+            if key == "message":
+                self.message = kwargs["message"]
+            else:
+                self.data[key] = kwargs[key]
```

Comment:
This allows attaching data to events. In the case of events JailExec we can read stdout and stderr from it when the event has ended.